# Hungarian translation for gdebi
# Copyright (c) 2007 Rosetta Contributors and Canonical Ltd 2007
# This file is distributed under the same license as the gdebi package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2007.
#
msgid ""
msgstr ""
"Project-Id-Version: gdebi\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2011-01-31 21:35+0100\n"
"PO-Revision-Date: 2009-11-13 19:58+0000\n"
"Last-Translator: Gergely Szarka <szarka.honved@gmail.com>\n"
"Language-Team: Hungarian <hu@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2011-06-08 15:28+0000\n"
"X-Generator: Launchpad (build 13144)\n"

#: ../gdebi:47
#, c-format
msgid ""
"usage: %prog [options] filename\n"
"For a graphical version run gdebi-gtk\n"
msgstr ""
"Használat: %prog [opciók] fájlnév\n"
"A grafikus változathoz futtassa a gdebi-gtk parancsot.\n"

#: ../gdebi:53 ../gdebi-gtk:51
msgid "Run non-interactive (dangerous!)"
msgstr "Futtatás nem interaktív módban (veszélyes!)"

#: ../gdebi:57
msgid "Set an APT configuration option"
msgstr "APT konfigurációs opció beállítása"

#: ../gdebi:61
msgid "Do not show progress information"
msgstr "Ne mutasson információt a folyamatról"

#: ../gdebi:65
msgid "Simulate only and print a apt-get install compatible line to stderr"
msgstr ""
"Csak szimulálja, és írjon ki egy apt-get install kompatibilis sort a "
"szabványos hibakimenetre"

#: ../gdebi:67
msgid "Use alternative root dir"
msgstr "Alternatív gyökérkönyvtár használata"

#: ../gdebi:75
#, c-format
msgid "gdebi error, file not found: %s\n"
msgstr "gdebi hiba, nem található a következő fájl: %s\n"

#: ../gdebi:94 ../gdebi:104
msgid "Need to be root to install packages"
msgstr "A csomagok telepítéséhez rendszergazdai jogosultság szükséges."

#: ../gdebi:106 ../GDebi/GDebiCli.py:154
msgid "Do you want to install the software package? [y/N]:"
msgstr "Kívánja telepíteni a szoftvercsomagot? [i/N]:"

#: ../gdebi-gtk:54
msgid "Auto close when the install is finished"
msgstr ""

#: ../gdebi-gtk:56
msgid "Use alternative datadir"
msgstr ""

#: ../gdebi-gtk:75
msgid "Software index is broken"
msgstr "A szoftverek tartalomjegyzéke sérült"

#: ../gdebi-gtk:76
msgid ""
"This is a major failure of your software management system. Please check for "
"broken packages with synaptic, check the file permissions and correctness of "
"the file '/etc/apt/sources.list' and reload the software information with: "
"'sudo apt-get update' and 'sudo apt-get install -f'."
msgstr ""
"Ez a szoftverkezelő rendszer súlyos hibáját jelzi. Ellenőrizze a törött "
"csomagokat a Synaptickal, ellenőrizze a fájl jogosultságait és az "
"/etc/apt/sources.list fájl helyességét, majd töltse újra a "
"szoftverinformációkat a \"sudo apt-get update\" és \"sudo apt-get install -"
"f\" parancsok kiadásával."

#: ../data/gdebi.ui.h:1
msgid "  "
msgstr "  "

#: ../data/gdebi.ui.h:2
msgid "<b><big>   </big></b>"
msgstr "<b><big> </big></b>"

#: ../data/gdebi.ui.h:3 ../GDebi/GDebiKDE.py:140
msgid "<b>Maintainer:</b>"
msgstr "<b>Karbantartó:</b>"

#: ../data/gdebi.ui.h:4 ../GDebi/GDebiKDE.py:141
msgid "<b>Priority:</b>"
msgstr "<b>Prioritás:</b>"

#: ../data/gdebi.ui.h:5 ../GDebi/GDebiKDE.py:142
msgid "<b>Section:</b>"
msgstr "<b>Csoport:</b>"

#: ../data/gdebi.ui.h:6 ../GDebi/GDebiKDE.py:143
msgid "<b>Size:</b>"
msgstr "<b>Méret:</b>"

#: ../data/gdebi.ui.h:7 ../GDebi/GDebiKDE.py:280
msgid "<b>To install the following changes are required:</b>"
msgstr "<b>A telepítéshez az alábbi módosítások szükségesek:</b>"

#: ../data/gdebi.ui.h:8 ../GDebi/GDebiKDE.py:139
msgid "<b>Version:</b>"
msgstr "<b>Verzió:</b>"

#: ../data/gdebi.ui.h:9
msgid "Automatically close after the changes have been successfully applied"
msgstr ""

#: ../data/gdebi.ui.h:10 ../GDebi/GDebiKDE.py:134
msgid "Description"
msgstr "Leírás"

#: ../data/gdebi.ui.h:11
msgid "Description:"
msgstr "Leírás:"

#: ../data/gdebi.ui.h:12 ../GDebi/GDebiKDE.py:133 ../GDebi/GDebiKDE.py:135
#: ../GDebi/GDebiKDE.py:281
msgid "Details"
msgstr "Részletek"

#: ../data/gdebi.ui.h:13
msgid "GPL, see /usr/share/common-licenses/GPL"
msgstr "GPL, lásd /usr/share/common-licenses/GPL"

#: ../data/gdebi.ui.h:14
msgid "Included files"
msgstr ""

#: ../data/gdebi.ui.h:15 ../data/gdebi.desktop.in.h:2
msgid "Install and view software packages"
msgstr "Szoftvercsomagok telepítése és megjelenítése"

#: ../data/gdebi.ui.h:16 ../data/gdebi.desktop.in.h:3
msgid "Package Installer"
msgstr "Csomagtelepítő"

#. first, we load all the default descriptions -- pyuic doesn't use
#. gettext as default (FIXME, copy code from language-selector)
#: ../data/gdebi.ui.h:17 ../GDebi/GDebiKDE.py:131
msgid "Package:"
msgstr "Csomag:"

#: ../data/gdebi.ui.h:18 ../GDebi/GDebiKDE.py:132
msgid "Status:"
msgstr "Státusz:"

#: ../data/gdebi.ui.h:19
msgid "Terminal"
msgstr "Terminál"

#: ../data/gdebi.ui.h:20
msgid "_Details"
msgstr "_Részletek"

#: ../data/gdebi.ui.h:21
msgid "_File"
msgstr "_Fájl"

#: ../data/gdebi.ui.h:22
msgid "_Help"
msgstr "_Súgó"

#. img = Gtk.Image()
#. img.set_from_stock(Gtk.STOCK_APPLY,Gtk.IconSize.BUTTON)
#. self.button_install.set_image(img)
#: ../data/gdebi.ui.h:23 ../GDebi/GDebi.py:320 ../GDebi/GDebi.py:360
msgid "_Install Package"
msgstr "_Csomag telepítése"

#: ../data/gdebi.ui.h:24
msgid "_Open..."
msgstr ""

#: ../data/gdebi.ui.h:25
msgid "_Refresh"
msgstr "_Frissítés"

#: ../data/gdebi.xml.in.h:1
msgid "Software package"
msgstr "Szoftvercsomag"

#: ../data/gdebi.desktop.in.h:1
msgid "GDebi Package Installer"
msgstr "GDebi csomagtelepítő"

#. Translators: it's for missing entries in the deb package,
#. e.g. a missing "Maintainer" field
#: ../GDebi/DebPackage.py:38
#, python-format
msgid "%s is not available"
msgstr "%s nem érhető el"

#: ../GDebi/GDebi.py:97
msgid "Loading..."
msgstr "Betöltés..."

#: ../GDebi/GDebi.py:168
msgid "Can not download as root"
msgstr "Nem tölthet le rootként"

#: ../GDebi/GDebi.py:169
msgid ""
"Remote packages can not be downloaded when running as root. Please try again "
"as a normal user."
msgstr ""
"Nem tölthet le távoli csomagokat, ha a programot root felhasználóként "
"futtatja. Próbálja újra normál felhasználóként."

#: ../GDebi/GDebi.py:182
msgid "Downloading package"
msgstr "Csomag letöltése"

#: ../GDebi/GDebi.py:189
msgid "Download failed"
msgstr "A letöltés meghiúsult"

#: ../GDebi/GDebi.py:190
#, python-format
msgid "Downloading the package failed: file '%s' '%s'"
msgstr "A csomag letöltése meghiúsult: „%s” fájl: „%s”"

#. set window title
#. set name
#: ../GDebi/GDebi.py:239 ../GDebi/GDebiKDE.py:181
#, python-format
msgid "Package Installer - %s"
msgstr "Csomagkezelő - %s"

#: ../GDebi/GDebi.py:296
msgid "Package control data"
msgstr ""

#: ../GDebi/GDebi.py:299
msgid "Upstream data"
msgstr ""

#: ../GDebi/GDebi.py:305
msgid "Error reading filelist"
msgstr ""

#: ../GDebi/GDebi.py:316
msgid "Error: "
msgstr "Hiba: "

#: ../GDebi/GDebi.py:332
msgid "Same version is already installed"
msgstr "Már telepítve van ugyanez a verzió"

#: ../GDebi/GDebi.py:333
msgid "_Reinstall Package"
msgstr "Csomag új_ratelepítése"

#: ../GDebi/GDebi.py:376
msgid "Selection is a directory"
msgstr ""

#: ../GDebi/GDebi.py:381 ../GDebi/GDebi.py:386
#, python-format
msgid "Error reading file content '%s'"
msgstr ""

#: ../GDebi/GDebi.py:390
msgid "File content can not be extracted"
msgstr ""

#: ../GDebi/GDebi.py:401
#, python-format
msgid "<b>To be removed: %s</b>"
msgstr "<b>Eltávolítandó: %s</b>"

#: ../GDebi/GDebi.py:403 ../GDebi/GDebiKDE.py:275
#, python-format
msgid "To be installed: %s"
msgstr "Telepítendő: %s"

#: ../GDebi/GDebi.py:418
msgid "Open Software Package"
msgstr "Szoftvercsomag megnyitása"

#: ../GDebi/GDebi.py:423
msgid "Software packages"
msgstr "Szoftvercsomagok"

#: ../GDebi/GDebi.py:452
msgid "File not found"
msgstr "A fájl nem található"

#: ../GDebi/GDebi.py:453
msgid "You tried to install a file that does not (or no longer) exist. "
msgstr ""
"Olyan fájlt próbált meg telepíteni, amely nem (vagy már nem) létezik. "

#. do it
#: ../GDebi/GDebi.py:463
msgid "Installing package file..."
msgstr "Csomag telepítése..."

#: ../GDebi/GDebi.py:465
msgid "Install unauthenticated software?"
msgstr "Kíván hitelesítés nélküli szoftvert telepíteni?"

#: ../GDebi/GDebi.py:466
msgid ""
"Malicious software can damage your data and take control of your system.\n"
"\n"
"The packages below are not authenticated and could therefore be of malicious "
"nature."
msgstr ""

#: ../GDebi/GDebi.py:493
msgid "You need to grant administrative rights to install software"
msgstr ""
"Adminisztrátori jogosultságokkal kell rendelkezni a szoftvercsomag "
"telepítéséhez"

#: ../GDebi/GDebi.py:494
msgid ""
"\n"
"It is a possible security risk to install packages files manually.\n"
"Install software from trustworthy software distributors only.\n"
msgstr ""
"\n"
"A csomagfájlok saját kezű telepítése biztonsági kockázatot jelent.\n"
"Csak megbízható szoftverterjesztőktől származó szoftvert telepítsen.\n"

#: ../GDebi/GDebi.py:517 ../GDebi/GDebi.py:578
msgid "Failed to install package file"
msgstr "A csomagfájl telepítése meghiúsult"

#: ../GDebi/GDebi.py:562 ../GDebi/GDebiKDE.py:327
msgid "Could not download all required files"
msgstr "Nem sikerült letölteni minden szükséges fájlt"

#: ../GDebi/GDebi.py:563 ../GDebi/GDebiKDE.py:328
msgid "Please check your internet connection or installation medium."
msgstr ""
"Kérem ellenőrizze az internetkapcsolatát vagy a telepítési adathordozót."

#: ../GDebi/GDebi.py:567 ../GDebi/GDebiKDE.py:332
msgid "Could not install all dependencies"
msgstr "Nem sikerült minden függőség telepítése"

#: ../GDebi/GDebi.py:568 ../GDebi/GDebiKDE.py:333
msgid ""
"Usually this is related to an error of the software distributor. See the "
"terminal window for more details."
msgstr ""
"Általában ez a szoftverterjesztő hibájával kapcsolatos. További részletekért "
"tekintse meg a terminálablakot."

#: ../GDebi/GDebi.py:584
#, python-format
msgid "Installing %s"
msgstr "%s telepítése"

#. self.label_action.set_markup("<b><big>"+_("Package installed")+"</big></b>")
#. show the button
#. self.button_deb_install_close.set_sensitive(True)
#. self.button_deb_install_close.grab_default()
#: ../GDebi/GDebi.py:600 ../GDebi/GDebiKDE.py:352
msgid "Installation finished"
msgstr "A telepítés befejeződött"

#: ../GDebi/GDebi.py:602
#, python-format
msgid "Package '%s' was installed"
msgstr "A következő csomag telepítésre került: %s"

#: ../GDebi/GDebi.py:604 ../GDebi/GDebiKDE.py:356
#, python-format
msgid "Failed to install package '%s'"
msgstr "Nem sikerült telepíteni a következő csomagot: „%s”"

#: ../GDebi/GDebi.py:606
msgid "Installation complete"
msgstr "A telepítés befejeződött"

#: ../GDebi/GDebi.py:613 ../GDebi/GDebiKDE.py:366
msgid "Failed to completely install all dependencies"
msgstr "Nem sikerült minden függőséget teljesen telepíteni"

#: ../GDebi/GDebi.py:614 ../GDebi/GDebiKDE.py:367
msgid "To fix this run 'sudo apt-get install -f' in a terminal window."
msgstr ""
"Ennek javításához futtassa a „sudo apt-get install -f” parancsot egy "
"terminálablakból."

#. ui
#: ../GDebi/GDebi.py:709 ../GDebi/KDEAptDialogs.py:73
#, python-format
msgid "Installing '%s'..."
msgstr "„%s” telepítése..."

#: ../GDebi/GDebi.py:808 ../GDebi/KDEAptDialogs.py:141
msgid "Installing dependencies..."
msgstr "Függőségek telepítése..."

#: ../GDebi/GDebi.py:853 ../GDebi/KDEAptDialogs.py:204
#: ../GDebi/KDEAptDialogs.py:217 ../GDebi/KDEAptDialogs.py:219
msgid "Downloading additional package files..."
msgstr "További csomagok letöltése..."

#: ../GDebi/GDebi.py:862 ../GDebi/KDEAptDialogs.py:217
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "%s. fájl, összesen %s. Sebesség: %sB/mp"

#: ../GDebi/GDebi.py:864 ../GDebi/KDEAptDialogs.py:219
#, python-format
msgid "File %s of %s"
msgstr "%s. fájl, összesen %s."

#. print "mediaChange %s %s" % (medium, drive)
#: ../GDebi/GDebi.py:871 ../GDebi/KDEAptDialogs.py:224
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Helyezze be a következő médiát: '%s', ebbe a meghajtóba: '%s'"

#: ../GDebi/GDebiCli.py:59
msgid "Configuration items must be specified with a =<value>\n"
msgstr "A konfigurációs elemeket =<érték> formában kell megadni\n"

#: ../GDebi/GDebiCli.py:65
#, python-format
msgid "Couldn't set APT option %s to %s\n"
msgstr "A(z) %s APT opció nem állítható be a következőre: %s\n"

#: ../GDebi/GDebiCli.py:78
#, python-format
msgid "Unknown package type '%s', exiting\n"
msgstr "Ismeretlen csomagtípus: „%s”; kilépés\n"

#: ../GDebi/GDebiCli.py:81
msgid "Failed to open the software package\n"
msgstr "A szoftvercsomag megnyitása meghiúsult\n"

#: ../GDebi/GDebiCli.py:82
msgid ""
"The package might be corrupted or you are not allowed to open the file. "
"Check the permissions of the file.\n"
msgstr ""
"A csomag sérült lehet vagy nincs joga a fájl megnyitásához. Ellenőrizze a "
"fájl jogosultságait.\n"

#: ../GDebi/GDebiCli.py:88
msgid "This package is uninstallable\n"
msgstr "Ez a csomag telepíthetetlen\n"

#: ../GDebi/GDebiCli.py:97 ../GDebi/GDebiKDE.py:217
msgid "No description is available"
msgstr "Nem érhető el leírás"

#: ../GDebi/GDebiCli.py:103
msgid "The following packages are UNAUTHENTICATED: "
msgstr "Az alábbi csomagok NINCSENEK HITELESÍTVE: "

#: ../GDebi/GDebiCli.py:107
msgid "Requires the REMOVAL of the following packages: "
msgstr "A következő csomagok ELTÁVOLÍTÁSA szükséges: "

#: ../GDebi/GDebiCli.py:112
msgid "Requires the installation of the following packages: "
msgstr "A következő csomagok telepítése szükséges: "

#: ../GDebi/GDebiCli.py:126 ../GDebi/GDebiCli.py:129
#, python-format
msgid "Error during install: '%s'"
msgstr ""

#: ../GDebi/GDebiKDE.py:136
msgid "Included Files"
msgstr "Telepített fájlok"

#: ../GDebi/GDebiKDE.py:138 ../GDebi/GDebiKDE.py:263
msgid "&Install Package"
msgstr "_Csomag telepítése"

#: ../GDebi/GDebiKDE.py:162
msgid "The package file does not exist"
msgstr "A csomagfájl nem létezik"

#: ../GDebi/GDebiKDE.py:163
msgid ""
"A nonexistent file has been selected for installation. Please select an "
"existing .deb package file."
msgstr ""
"Egy nem létező fájl került kiválasztásra. Egy létező .deb csomagfájlt "
"válasszon ki."

#. self.textLabel1_3_2.setText(_("Same version is already installed"))
#: ../GDebi/GDebiKDE.py:231
msgid "&Reinstall Package"
msgstr "Csomag új_ratelepítése"

#: ../GDebi/GDebiKDE.py:266
msgid "Re&install Package"
msgstr "Csomag új_ratelepítése"

#: ../GDebi/GDebiKDE.py:277
#, python-format
msgid "To be removed: %s"
msgstr "Eltávolítandó: %s"

#: ../GDebi/GDebiKDE.py:354
msgid "<b>"
msgstr "<b>"

#: ../GDebi/GDebiCommon.py:67
msgid "Broken dependencies"
msgstr "Törött függőségek"

#: ../GDebi/GDebiCommon.py:68
msgid ""
"Your system has broken dependencies. This application can not continue until "
"this is fixed. To fix it run 'gksudo synaptic' or 'sudo apt-get install -f' "
"in a terminal window."
msgstr ""
"A rendszeren törött függőségek vannak. Az alkalmazás a javításig nem képes "
"folytatni a munkát. Próbálja meg terminálablakban futtatni a „gksudo "
"synaptic” vagy „sudo apt-get install -f” parancsok egyikét."

#: ../GDebi/GDebiCommon.py:85
#, python-format
msgid "'%s' is not a Debian package"
msgstr "„%s” nem Debian csomagfájl"

#: ../GDebi/GDebiCommon.py:86
#, python-format
msgid ""
"The MIME type of this file is '%s' and can not be installed on this system."
msgstr "A fájl „%s” MIME típusú, emiatt nem telepíthető erre a rendszerre."

#: ../GDebi/GDebiCommon.py:90
#, python-format
msgid "Could not open '%s'"
msgstr "Nem nyitható meg a következő: „%s”"

#: ../GDebi/GDebiCommon.py:91
msgid ""
"The package might be corrupted or you are not allowed to open the file. "
"Check the permissions of the file."
msgstr ""
"A csomag sérült lehet vagy nincs joga a fájl megnyitásához. Ellenőrizze a "
"fájl jogosultságait."

#: ../GDebi/GDebiCommon.py:107
msgid "Same version is available in a software channel"
msgstr "Ugyanez a verzió elérhető egy szoftvercsatornán át is"

#: ../GDebi/GDebiCommon.py:108
msgid "You are recommended to install the software from the channel instead."
msgstr "A csomag telepítése inkább a szoftvercsatornán keresztül ajánlott."

#: ../GDebi/GDebiCommon.py:112
msgid "An older version is available in a software channel"
msgstr "Egy régebbi verzió érhető el egy szoftvercsatornán keresztül."

#: ../GDebi/GDebiCommon.py:113
msgid ""
"Generally you are recommended to install the version from the software "
"channel, since it is usually better supported."
msgstr ""
"A szoftvercsatornából származó csomagverzió telepítése ajánlott, mivel az "
"általában jobban támogatott."

#: ../GDebi/GDebiCommon.py:118
msgid "A later version is available in a software channel"
msgstr "Egy újabb verzió érhető el egy szoftvercsatornán keresztül."

#: ../GDebi/GDebiCommon.py:120
msgid ""
"You are strongly advised to install the version from the software channel, "
"since it is usually better supported."
msgstr ""
"Erősen ajánlott a szoftvercsatornából származó csomagverzió telepítése, "
"mivel az általában jobban támogatott."

#: ../GDebi/GDebiCommon.py:128
msgid "All dependencies are satisfied"
msgstr "Minden függőség kielégítve"

#. FIXME: use ngettext here
#: ../GDebi/GDebiCommon.py:131
#, python-format
msgid "Requires the <b>removal</b> of %s packages\n"
msgstr "%s csomag <b>eltávolítását</b> igényli\n"

#: ../GDebi/GDebiCommon.py:133
#, python-format
msgid "Requires the installation of %s packages"
msgstr "%s csomag telepítését igényli"

#: ../GDebi/GDebiCommon.py:141 ../GDebi/GDebiCommon.py:159
msgid "Only one software management tool is allowed to run at the same time"
msgstr "Csak egy szoftverkezelő eszköz futhat egyidejűleg"

#: ../GDebi/GDebiCommon.py:143 ../GDebi/GDebiCommon.py:161
msgid ""
"Please close the other application e.g. 'Update Manager', 'aptitude' or "
"'Synaptic' first."
msgstr ""
"Először zárja be a másik alkalmazást (például: Frissítéskezelő, aptitude, "
"Synaptic)."

#. change = QMessageBox.question(None, _("Media Change"), msg, QMessageBox.Ok, QMessageBox.Cancel)
#: ../GDebi/KDEAptDialogs.py:226
msgid "Media Change"
msgstr "Adathordozó-csere"

#~ msgid "gdebi"
#~ msgstr "gdebi"

#, python-format
#~ msgid "Wrong architecture '%s'"
#~ msgstr "Rossz architektúra: \"%s\""

#~ msgid "Y"
#~ msgstr "I"

#~ msgid "Open..."
#~ msgstr "Megnyitás..."

#~ msgid ""
#~ "Malicous software can damage your data and take control of your system.\n"
#~ "\n"
#~ "The packages below are not authenticated and could therefor be of malicous "
#~ "nature."
#~ msgstr ""
#~ "A rosszindulatú szoftverek tönkretehetik az adatait és átvehetik a rendszere "
#~ "feletti ellenőrzést.\n"
#~ "\n"
#~ "Az alábbi csomagok hitelesítés nélküliek és ennek megfelelően akár "
#~ "rosszindulatúak is lehetnek."

#, python-format
#~ msgid "Dependency is not satisfiable: %s\n"
#~ msgstr "A következő függőség nem elégíthető ki: %s\n"

#~ msgid "Analysing dependencies"
#~ msgstr "Függőségek elemzése"

#, python-format
#~ msgid "Conflicts with the installed package '%s'\n"
#~ msgstr "Ütközés a(z) '%s' telepített csomaggal\n"

#~ msgid "No Architecture field in the package"
#~ msgstr "A csomagban nincs architektúra mező"

#~ msgid "A later version is already installed"
#~ msgstr "Már telepítve van egy újabb verzió"

#, python-format
#~ msgid "Breaks exisiting package '%s' conflict: %s (%s %s)\n"
#~ msgstr "Megsérti a meglévő „%s” csomag függőségeit; ütközés: %s (%s %s)\n"

#, python-format
#~ msgid "Breaks exisiting package '%s' dependency %s (%s %s)\n"
#~ msgstr "Megsérti a meglévő „%s” csomag %s függőségét (%s %s)\n"

#, python-format
#~ msgid "Cannot install '%s'"
#~ msgstr "„%s” nem telepíthető"

#~ msgid "Failed to satisfy all dependencies (broken cache)"
#~ msgstr "Nem sikerült kielégíteni minden függőséget (a gyorsítótár sérült)"

#, python-format
#~ msgid "IOError during filelist read: %s"
#~ msgstr "IOError a fájllista olvasása során: %s"

#~ msgid "List of files could not be read, please report this as a bug"
#~ msgstr ""
#~ "A fájlok listáját nem sikerült beolvasni, kérjük jelezze ezt a hibát a "
#~ "fejlesztőknek."
